src/components/tooltip/Tooltip.stories.jsx

```diff
@@ -1,156 +1,78 @@
-                textAlign: "center",
-              }}
-            >
-              Tooltips
-            </h1>
-            <div
-              style={{
-                border: "1px solid grey",
-                padding: "1rem",
-                display: "flex",
-                gap: 32,
-                justifyContent: "space-between",
-                margin: 32,
-              }}
-            >
-              <Tooltip content={linkContent} supressArrow>
-                Hover me to show a link
-              </Tooltip>
-              <Tooltip content={buttonContent} supressArrow>
-                Hover me to show a button
-              </Tooltip>
-            </div>
-          </div>
-    
-          <div
-            style={{
-              position: "relative",
-              display: "flex",
-              justifyContent: "center",
-              alignItems: "center",
-              flexWrap: "wrap",
-              gap: 32,
-              height: 300,
-              width: 450,
-              border: "1px solid grey",
-              padding: "1rem",
-            }}
-          >
-            <div
-              style={{
-                bottom: "1rem",
-                position: "absolute",
-                ...buttonContainerStyle,
-              }}
-            >
-              <Tooltip placement="top" content={tooltipContent}>
-                Top
-              </Tooltip>
-            </div>
-            <div
-              style={{
-                right: "1rem",
-                flexDirection: "column",
-                textAlign: "right",
-                position: "absolute",
-                ...buttonContainerStyle,
-              }}
-            >
-              <Tooltip placement="left" content={tooltipContent}>
-                Left
-              </Tooltip>
-            </div>
-            <div
-              style={{
-                left: "1rem",
-                flexDirection: "column",
-                position: "absolute",
-                ...buttonContainerStyle,
-              }}
-            >
-              <Tooltip placement="right" content={tooltipContent}>
-                Right
-              </Tooltip>
-            </div>
-            <div
-              style={{
-                top: "1rem",
-                position: "absolute",
-                ...buttonContainerStyle,
-              }}
-            >
-              <Tooltip placement="bottom" content={tooltipContent}>
-                Bottom
-              </Tooltip>
-            </div>
-          </div>
-        </div>
-      );
+        flexDirection: "column",
+        gap: 4,
+      }}
+    >
+      <h1 style={{ fontSize: 14, fontWeight: "bold", textAlign: "center" }}>A friendly reminder</h1>
+      <p
+        style={{
+          fontSize: 12,
+          lineHeight: "16px",
+          textAlign: "center",
+          fontWeight: "lighter",
+        }}
+      >
+        You can save your search view and come back to it later just in one click
```

Comment:
@nicolasshiken Please add a story with an check icon like in designs for testing.